R/accounts.R

```diff
@@ -294,6 +294,12 @@ findAccountInfo <- function(name = NULL, server = NULL, error_call = caller_env(
 
   accountDcf <- read.dcf(configFile, all = TRUE)
   info <- as.list(accountDcf)
+
+  # Account records previously had username, now have name. Internal callers expect "name", but
+  # external callers may expect "username". (#1024)
+  info$name <- info$name %||% info$username
+  info$username <- info$username %||% info$name
```

Comment:
```suggestion
  info$name <- info$name %||% info$username
  info$username <- info$name
```

Makes it a little more clear they're always the same IMO.

Reply:
Done.